HISTORY.md

```diff
@@ -1,5 +1,8 @@
 # History
 
+## 0.2.3 (2018-10-18)
+	* Creates subdirectory structure
```

Comment:
this should be `0.2.0`, when you increment the minor version, the patch version gets reset